public/content/documentation/native/controls/expandable.md

```diff
@@ -0,0 +1,253 @@
+## General Notes
+
+How to test an expandable region
+
+## Videos
+
+### iOS Voiceover
+<video controls>
+  <source src="media/video/native/expandable/expandable-iOSVoiceOver.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Android Talkback
+<video controls>
+  <source src="media/video/native/expandable/expandable-AndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test an expandable region
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, arrow keys or ctl+tab: Focus visibly moves to the expandable region
+   - Spacebar: Activates on iOS and Android
+   - Enter: Activates on Android
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the element, expresses its name, role (state, if applicable)
+   - Doubletap: Activates the expandable region
+
+3. Listen to screenreader output on all devices
+
+   - Name: Purpose is clear and matches visible label
+   - Role: Identifies as an expandable region in iOS and expandable region or "double tap to activate" in Android
+   - Group: Visible label is grouped or associated with the expandable region in a single swipe
```

Comment:
I'm only seeing "Name" and "Role" listed on step 3 of condensed?

Reply:
#a11y - Native Accessibility Acceptance Criteria

How to test a step indicator

1. Test keyboard actions
- Tab, arrow keys or ctl+tab: Focus visibly moves to the part of the control that is interactive
- Spacebar: Activates on iOS and Android
- Enter: Activates on Android
      

2. Test mobile screenreader gestures
- Swipe: Focus moves to the control, expresses its name, role, state (if applicable)
- Doubletap: Activates the control

3. Listen to screenreader output on all devices
- Name: Purpose is clear (ex: "Step 6 of 7, Account")
- Role: Identifies itself as a button or dropdown

4. Device OS settings
- Text resize: Text can resize up to 200% without losing information

Full information: https://www.magentaa11y.com/checklist-native/step-indicator/